Microsoft.Toolkit.Uwp.UI/Extensions/DependencyObjectExtensions.cs

```diff
@@ -0,0 +1,487 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+#nullable enable
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="DependencyObject"/> type.
+    /// </summary>
+    public static class DependencyObjectExtensions
```

Comment:
Miss a move here? Also, since these are all the Visual Tree related ones, should we have the file be `DependencyObjectExtensions.VisualTree.cs` so it's more easily identifiable/findable?

Reply:
I'm 100% sure I did use `giv mv` for every single moved file in this whole PR, that's super weird.
Just double checked, all the commits do look fine on my end:

![image](https://user-images.githubusercontent.com/10199417/107362373-f45abe80-6ad8-11eb-84ca-a5f1c9db7c35.png)

Any ideas why could GitHub be reporting this incorrectly on the website? 🤔

Reply:
It's probably because the other PR isn't merged yet and this is still basing off of main vs. the other PR...